CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 10.0.0-BETA.6 (YYYY-MM-DD)
+
+We no longer support Realm Cloud (legacy), but instead the new MongoDB Realm Cloud. MongoDB Realm is a serverless platform that enables developers to quickly build applications without having to set up server infrastructure. MongoDB Realm is built on top of MongoDB Atlas, automatically integrating the connection to your database.
+
+The old Realm Cloud legacy API's have undergone significant refactoring. The new API's are all located in the `io.realm.mongodb` package with `io.realm.mongodb.App` as the entry point.
```

Comment:
I was just being curious about the Saxon genitive in `API's` and stumbled upon the NY Times style guideline on acronym plurals: https://afterdeadline.blogs.nytimes.com/2010/04/13/faqs-on-style/#:~:text=Mind%20your%20p's%20and%20q's,1990s%2C%20747s%2C%20size%207s.

```suggestion
The old Realm Cloud legacy APIs have undergone significant refactoring. The new APIs are all located in the `io.realm.mongodb` package with `io.realm.mongodb.App` as the entry point.
```

Reply:
My English grammar has a lot of holes 😄 🙈